src/rgw/rgw_common.cc

```diff
@@ -201,7 +201,7 @@ static string get_abs_path(const string& request_uri) {
   return request_uri.substr(beg_pos, len - beg_pos);
 }
 
-req_info::req_info(CephContext *cct, class RGWEnv *e) : env(e) {
+req_info::req_info(CephContext *cct, class RGWEnv *_env) : env(_env) {
```

Comment:
The underscore in `_env` [may be omitted](https://stackoverflow.com/a/6185043):
```cpp
req_info::req_info(CephContext *cct, class RGWEnv *env) : env(env) {
```
A bit more const-correct variant would look like:
```cpp
req_info::req_info(CephContext* const cct, class RGWEnv* const env) : env(env) {
```